src/context/wishlist-context.js

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+import { createContext, useContext, useReducer } from "react";
+import { wishlistReducer } from "../reducer/wishlistReducer";
+
+const initialState = {
+    wishlist:[],
+    totalItems:0
```

Comment:
You can find the total number of items on the fly in whichever component you want using wishlist.length . I dont think you need to store it seperately here.